kusto/test/etoe/etoe_test.go

```diff
@@ -490,6 +502,33 @@ func TestFileIngestion(t *testing.T) {
 			},
 			want: mockRows,
 		},
+		{
+			desc: "Ingestion to two different clusters",
+			src:  createCsvFileFromData(mockRows),
+			stmt: kusto.NewStmt("Logs | order by header_api_version asc"),
+			setup: func() error {
+				err := createIngestionTable(client, testConfig.Database)
```

Comment:
Do you delete these tables after?

Reply:
Currently no. Also the tables don't have unique names per test.

These are both fixed in the StreamingIngest branch though, so once we'll merge it will be ok.